DataConnectors/ESET Enterprise Inspector/azuredeploy_ESETEnterpriseInspector_API_FunctionApp.json

```diff
@@ -178,7 +178,7 @@
                         "domainLogin": "[parameters('domainLogin')]",
                         "verifySsl": "[parameters('verifySsl')]",
                         "startFromID": "[parameters('startFromID')]",
-                        "WEBSITE_RUN_FROM_PACKAGE": "https://aka.ms/sentinelesetenterpriseinspectorfunctionapp"
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://aka.ms/sentinel-esetenterpriseinspector-functionapp"
```

Comment:
this redirects here: https://raw.githubusercontent.com/dmaasland/Azure-Sentinel/master/DataConnectors/ESET%20Enterprise%20Inspector/AzureFunctionESETEnterpriseInspector.zip, 

which is not master branch, can you double check?

Reply:
Resolved with link pointing to master, updated the link in aka.ms.